build fail looks real
```
======================================================================
ERROR: corehq.apps.userreports.tests.test_pillow:AsyncIndicatorTest.test_async_save_fails
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/mock/mock.py", line 1297, in patched
    arg = patching.__enter__()
  File "/vendor/lib/python2.7/site-packages/mock/mock.py", line 1369, in __enter__
    original, local = self.get_original()
  File "/vendor/lib/python2.7/site-packages/mock/mock.py", line 1343, in get_original
    "%s does not have the attribute %r" % (target, name)
AttributeError: <module 'corehq.apps.userreports.tasks' from '/mnt/commcare-hq-ro/corehq/apps/userreports/tasks.pyc'> does not have the attribute u'_get_config'
-------------------- >> begin captured logging << --------------------
```

--- a/corehq/apps/userreports/tests/test_pillow.py
+++ b/corehq/apps/userreports/tests/test_pillow.py
@@ -445,7 +445,7 @@ class AsyncIndicatorTest(TestCase):
             self.assertEqual(errors.count(), 0)
             self.assertEqual(indicators.count(), 0)
 
-    @patch('corehq.apps.userreports.tasks._get_config')
+    @patch('corehq.apps.userreports.tasks._get_config_by_id')
     def test_async_save_fails(self, config):
         # process_changes will generate an exception when trying to use this config
         config.return_value = None

